core/unit_test/TestTeamVector.hpp

```diff
@@ -864,6 +864,88 @@ class TestTripleNestedReduce {
 
 #endif
 
+template <typename ExecutionSpace, bool bInclusive, class Reducer>
```

Comment:
No Hungarian notation.  Either go for `IsInclusive` or (better) use a scoped enumeration `enum class ScanType : bool { Inclusive, Exclusive };`